src/styles/blocks/day.scss

```diff
@@ -0,0 +1,23 @@
+.day {
+  background-color: #eee;
+  border: 1px solid black;
+  width: $calendar-item-width;
+  height: $calendar-item-height;
+  position: relative;
```

Comment:
Positioning should be in .calendar__day if you need it there.

Reply:
done